Changed pubspec.yaml? Restart program on device. (#6045)
flutter/website: src/docs/development/ui/layout/tutorial.md

    as shown above.
  * The pubspec file is also sensitive to white
    space, so use proper indentation.

>   * You might need to restart the running program
>     (in simulation or on connected device) for the
>     changes in the file to take effect.

Reviewer: ```suggestion
  * You might need to restart the running program
    (on the simulator or a connected device) for the
    pubspec changes to take effect.
```
